### ISSUE TITLE
BaseTransformationControllers do not respect isEnabled on TransformableNode

### ISSUE THREAD

**Issue by rubixhacker** [google-ar/sceneform-android-sdk#25]
I am trying to disable scale and rotation on TransformableNode by doing the following

```
val node = TransformableNode(fragment.transformationSystem).apply {
            translationController.isEnabled = true
            rotationController.isEnabled = false
            scaleController.isEnabled = false
            renderable = modelRenderable
            setParent(anchorNode)
        }
```

However the node is still scalable and rotatable by the user.

**Comment by malik-at-work** [comment]
I took a look at this and this doesn't seem quite right.  I'll update with more info as I get it.

**Comment by rubixhacker** [comment]
Thanks, that resolved the issue!

**Comment by malik-at-work** [comment]
Fixed in Sceneform 1.3
